exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/_metric_exporter/__init__.py

```diff
@@ -168,7 +199,12 @@ def _translate_data(
             )
         )
 
-    def export(self, metrics: Sequence[Metric]) -> MetricExportResult:
+    def export(
+        self,
+        metrics: Dict[
```

Comment:
I see this annotation repeated a lot. It would be good to make a type alias instead

Reply:
Added a type alias.